tools/ci-build.sh

```diff
@@ -39,7 +39,7 @@ P4C_DIR=$(readlink -f ${THIS_DIR}/..)
 # Only execute the steps necessary to successfully run CMake.
 : "${CMAKE_ONLY:=OFF}"
 # The build generator to use. Defaults to Ninja. If that is not available "make" is used.
-: "${BUILD_GENERATOR:=Ninja}"
+: "${BUILD_GENERATOR:="Unix Makefiles"}"
```

Comment:
You probably need to enclose the inner string in apostrophes:

```suggestion
: "${BUILD_GENERATOR:='Unix Makefiles'}"

Reply:
This does seem to break the script.